src/wp-includes/l10n.php

```diff
@@ -1154,6 +1159,10 @@ function load_script_textdomain( $handle, $domain = 'default', $path = '' ) {
 	$file_base       = 'default' === $domain ? $locale : $domain . '-' . $locale;
 	$handle_filename = $file_base . '-' . $handle . '.json';
 
+	if ( ! $path ) {
+		$path = $wp_textdomain_registry->get( $domain, $locale );
```

Comment:
I was wrong about the slash being omitted. We do need `untrailingslashit` here after all.

Reply:
got it, I will move this if aboce the already existing 
```php
$path   = untrailingslashit( $path );
```

Reply:
Looking at the code, I see that the trailing slash is always present in a path returned by `wp_textdomain_registry->get`. The `set` method calls `rtrim( $path, '/' ) . '/'` when setting the path. And the `_load_textdomain_just_in_time` function relies on the slash being there, building a file path with:
```php
$mofile = "{$path}{$locale}.mo";
```

Reply:
@jsnajdr Not sure I follow...? That's why I asked to add back the `untrailingslashit` here.

Reply:
Oh nevermind, what I meant that the registry path doesn't need to be "sanitized", like the `$path` argument needs to be, because the registry path always has the slash. There is no uncertainty in the input, but of course `untrailingslashit` is useful even without uncertainty. I don't have any further feedback on this PR 🙂